src/universal/components/Dashboard/DashModal.js

```diff
@@ -107,19 +104,19 @@ const styleThunk = (theme, props) => ({
     animationName: animateOut
   },
 
-  viewport: {
-    left: 0
-  },
-
-  main: {
+  [ui.modalLayout[0]]: {
```

Comment:
if we write the type out instead of referencing an array this would be easier to read